apps/web/layout/components/DashNav.tsx

```diff
@@ -3,10 +3,10 @@ import Link from 'next/link';
 import { FiArrowLeft, FiHome, FiUser } from 'react-icons/fi';
 
 export const DashNav = () => (
-    <div className="flex flex-col items-center bg-[#f9f5ff] w-[220px] p-9 pt-[64px] min-h-full">
+    <div className="flex flex-col items-center bg-[#f9f5ff] w-[220px] p-9 pt-[64px] h-[100vh]">
```

Comment:
use h-screen instead of h-[100vh]

Don't hardcode colors. Instead put them in tailwind config